### PR TITLE
Dialog design has been improved

### LINKED ISSUE #1900: The design of the dialog box is inappropriate
**Summary:** 

The design of the dialog box is inappropriate.

**Steps to reproduce:** 

demo

**Device and Android version:** 

Device : OnePlus Nord
Android 12

### PR DESCRIPTION
**Pull request by Aditya-gupta99**
Fixes #1900 

https://user-images.githubusercontent.com/94394661/234108301-a02124c5-92c8-4b22-9ef0-3f4eea9d703e.mp4

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Aditya-gupta99** [comment]
Replacing PR #1901 with this one.

**Comment by Aditya-gupta99** [comment]
@avinash14022002 Please review this PR.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java [hunk 1]
```diff
@@ -9,16 +9,17 @@
...
 import android.view.View;
 import android.view.inputmethod.InputMethodManager;
 import android.widget.Toast;
 
+import androidx.appcompat.widget.SwitchCompat;
+import androidx.appcompat.widget.Toolbar;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentTransaction;
+
```
Comment (avinash14022002) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/core/MifosBaseActivity.java:22]: adding extra line change in codebase please keep it as it

**Review by avinash14022002 — CHANGES_REQUESTED** [review]
please do the needful changes

**Comment by avinash14022002** [comment]
> please do the needful changes

@Aditya-gupta99 
solve the merge conflict

**Pull request closed without merging**